pymess/webhooks/mandrill.py

```diff
@@ -42,6 +40,6 @@ def post(self, request, *args, **kwargs):
     def process_event(self, event_dict):
         message_id = event_dict.get('_id', None)
         if message_id:
-            message = get_object_or_none(EmailMessage, extra_sender_data__contains='"_id":"{}"'.format(message_id))
```

Comment:
Huh, apparently the JSON field is being treated as text... I think that proper JSON querying might help, I don't understand why it was not used, maybe the leading underscore causes the problem?

Reply:
It is not JSON field in database